src/main/kotlin/uk/gov/justice/digital/hmpps/hmppsmanageadjudicationsapi/services/DraftAdjudicationService.kt

```diff
@@ -44,6 +47,13 @@ class DraftAdjudicationService(
         dateTimeOfIncident = dateTimeOfIncident,
         handoverDeadline = dateCalculationService.calculate48WorkingHoursFrom(dateTimeOfIncident)
       ),
+      // Temporary code for backwards compatibility
+      incidentRole = incidentRole?.let {
+        IncidentRole(
+          roleCode = incidentRole.roleCode,
+          associatedPrisonersNumber = incidentRole.associatedPrisonersNumber,
+        )
+      } ?: IncidentRole(roleCode = null, associatedPrisonersNumber = null),
```

Comment:
Temporary code - to be removed in next PR

Reply:
Will we need to blat the data that is already in the database - i.e. any draft adjudications without an IncidentRole?